### PR TITLE
#36 Introduced `series` with `max` and `min`

### PR DESCRIPTION
**Pull request by maxonfjvipon**
Closes: #36

### CODE REVIEW

**Comment by maxonfjvipon** [comment]
@Graur could you please take a look? `max` and `min` objects and their tests are almost identical but some of them work, some don't.

**Comment by maxonfjvipon** [comment]
@Graur please, take a look

File: src/main/eo/org/eolang/math/series.eo [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  # @todo #36:30 min. The array may consist of objects of any type. `max` and
+  #  `min` objects should work only with an array of numbers.
+  # Find max element in the array
+  [] > max
+    list ^ > self-as-list!
+
+    if. > @
+      self-as-list.is-empty
+      []
+        "Can't get max of empty array" > msg
```
Comment (Graur) [inline src/main/eo/org/eolang/math/series.eo:42]: @maxonfjvipon I think we can use [error](https://news.eolang.org/2022-07-18-error-and-try-catch.html) object here

File: src/main/eo/org/eolang/math/series.eo [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            max
+
+  # Find min element in the array
+  [] > min
+    list ^ > self-as-list!
+
+    if. > @
+      self-as-list.is-empty
+      []
+        "Can't get min of empty array" > msg
```
Comment (Graur) [inline src/main/eo/org/eolang/math/series.eo:59]: @maxonfjvipon Same here, we can use `error` object

**Review by Graur — CHANGES_REQUESTED** [review]
@maxonfjvipon Good one! Just a few comments from my side:

File: src/main/eo/org/eolang/math/series.eo (hunk 1 above)
Comment (maxonfjvipon) [reply src/main/eo/org/eolang/math/series.eo]: Done!

**Comment by maxonfjvipon** [comment]
@Graur I think it's good to merge

**Review by Graur — APPROVED** [review]
@maxonfjvipon Thanks!
@yegor256 please, take a look

**Comment by yegor256** [comment]
@rultor merge

**Comment by maxonfjvipon** [comment]
@yegor256 please, try to merge one more time

**Comment by yegor256** [comment]
@rultor merge

**Comment by maxonfjvipon** [comment]
@Graur is there a syntax mistake in `@todo`? PDD issue was not created for some reason

**Comment by Graur** [comment]
@maxonfjvipon Looks good. Maybe next merge published them

**Pull request merged**